### PR TITLE
doc(Apache APISIX): add a description about Apache APISIX.

### PR DESCRIPTION
**Pull request by Chever-John**
Closes #<issue number>

## Description

I'm from the Apache APISIX community and would like to add a paragraph to the official GraphQL website. As a cloud-native API gateway, Apache APISIX has been designed from the beginning with the ability to identify matches for GraphQL syntax. By effectively matching GraphQL statements carried in requests, it can filter out abnormal traffic to further ensure security and improve system performance.

We want to add a paragraph. Thanks ~

### CODE REVIEW

File: src/content/code/services/apisix.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: Apache APISIX
+description: Apache APISIX is a dynamic, real-time, high-performance API gateway. APISIX provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, etc. As a cloud-native API gateway, Apache APISIX already can recognize GraphQL syntax at the beginning of its design. Efficiently matching GraphQL statements carried in requests can filter out abnormal traffic to further ensure security and improve system performance. For more information, go to https://apisix.apache.org/blog/2022/03/02/
```
Comment (juzhiyuan) [inline src/content/code/services/apisix.md]:
```suggestion
description: Apache APISIX is a dynamic, real-time, high-performance API gateway. APISIX provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, etc. As a cloud-native API gateway, Apache APISIX already can recognize GraphQL syntax at the beginning of its design. Efficiently matching GraphQL statements carried in requests can filter out abnormal traffic to further ensure security and improve system performance. For more information, please visit [How to Use GraphQL with API Gateway Apache APISIX](https://apisix.apache.org/blog/2022/03/02/apisix-integration-graphql-plugin/)
```

Comment (pottekkat) [inline src/content/code/services/apisix.md]: The ideal description length is 70-155 characters. This is just for discoverability and SEO. So, we can make it much smaller.

**Review by pottekkat — CHANGES_REQUESTED** [review]

File: src/content/code/services/apisix.md (hunk 1 above)
Comment (Chever-John) [reply src/content/code/services/apisix.md]: Sure~
Let me try

Comment (orta) [reply src/content/code/services/apisix.md]: Structurally this all looks good - just let me know when you're happy

Comment (Chever-John) [reply src/content/code/services/apisix.md]: Solved

Comment (Chever-John) [reply src/content/code/services/apisix.md]: @orta I think I'm ready to go. Please review, sir.
Thanks first.

**Comment by orta** [comment]
Cool, thanks folks

**Pull request merged**